build.cake

```diff
@@ -0,0 +1,268 @@
+#tool nuget:?package=NUnit.ConsoleRunner&version=3.4.1
```

Comment:
If this directive doesn't specify a version, does it default to latest? That would be one less thing to update if we could avoid it!